rosys/vision/rtsp_camera/rtsp_camera.py

```diff
@@ -109,23 +109,24 @@ async def capture_image(self) -> None:
 
     def set_fps(self, fps: int) -> None:
         assert self.device is not None
-        assert self.device.settings_interface is not None
-        self.device.settings_interface.set_fps(stream_id=self.jovision_profile, fps=fps)
+
+        self.device.set_fps(fps)
         self.polling_interval = 1.0 / fps
 
     def get_fps(self) -> int | None:
         assert self.device is not None
-        assert self.device.settings_interface is not None
-        fps = self.device.settings_interface.get_fps(stream_id=self.jovision_profile)
-        return fps
+
+        return self.device.get_fps()
 
     def set_jovision_profile(self, profile: int) -> None:
         assert self.device is not None
-        self.jovision_profile = profile
+
```

Comment:
Don't we need to update `self.jovision_profile` here?

Reply:
You're right, the parameter is redundant and should be removed